webApp/src/main/java/com/infoshare/register/RegisterService.java

```diff
@@ -0,0 +1,66 @@
+package com.infoshare.register;
+
+import com.infoshare.services.UserService;
+import domain.activities.Activity;
+import domain.activities.ActivityLevel;
+import domain.activities.SportDisciplines;
+import domain.location.Address;
+import domain.location.Town;
+import domain.users.Gender;
+import domain.users.User;
+import domain.users.UserDTO;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Component
+public class RegisterService {
+
+    private static final String MENU_HEADER = "Welcome in *** GymBuddy Finder ***";
+
+    public String getMenuHeader() {
+        return MENU_HEADER;
+    }
+
```

Comment:
Wydaje mi się, że stałą MENU_HEADER można usunąć jak i również metodę getMenuHeader(),
w związku z tym, że mamy widoki porobione, to nie są używane